test/integration/kube_api_discovery_test.go

```diff
@@ -430,10 +430,14 @@ func TestCRDAdditionalPrinterColumns_Parallel(t *testing.T) {
 		addSuffix("credentialissuers.config.concierge"): {
 			"v1alpha1": []apiextensionsv1.CustomResourceColumnDefinition{
 				{Name: "ProxyMode", Type: "string", JSONPath: ".spec.impersonationProxy.mode"},
-				// CredentialIssuers status is a list of strategies, each with its own status. Unfortunately,
+				// CredentialIssuer status is a list of strategies, each with its own status. Unfortunately,
 				// AdditionalPrinterColumns cannot show multiple results, e.g. a list of strategy types where
 				// the status is equal to Successful. See https://github.com/kubernetes/kubernetes/issues/67268.
-				// So we don't show any status as a AdditionalPrinterColumn at the moment.
+				// Although this selector can evaluate to multiple results, the kubectl implementation of JSONPath
+				// will always only show the first result. Thus, this column will show the first successful strategy
```

Comment:
The kubectl code can handle arrays fine.  It is the server side CRD logic that does not support it.

https://github.com/kubernetes/kubernetes/blob/c4587a62f9a29d4de4a115fac84ad93bd8f510ca/staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresource/tableconvertor/tableconvertor.go#L116-L117

Reply:
(The server generates the table output, not the client)

Reply:
Updated!